app/src/ui/changes/files-changed-badge.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+
+interface IFilesChangedBadgeProps {
+  readonly numFilesChanged: number
+}
+
+/** The number that can be displayed as a specific value */
+const LargeNumber = 300
```

Comment:
This is kind of the maximum value we display here, right? Could we make this a bit more descriptive, something like `MaximumChangesCount`?